pierce-portfolio/portfolio/src/main/webapp/fotos.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+   <head>
+    <meta charset="UTF-8">
+    <title>Pierce D. Wilson</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+ <div class="topnav">
+    <a href="/">About me</a>
+    <a href="/resume.html">Resume</a>
+    <!-- <a href="/projects.html">Projects</a> -->
```

Comment:
Please delete commented out lines if you're not using here and below.

Reply:
Okay